abstract.md

```diff
@@ -4,3 +4,5 @@ Abstract
 This document, “Guidance on Applying WCAG 2.2 to Non-Web Information and Communications Technologies (WCAG2ICT)” describes how the Web Content Accessibility Guidelines (WCAG) 2.2 \[\[WCAG22\]\] and its principles, guidelines, and success criteria can be applied to non-web Information and Communications Technologies (ICT), specifically to non-web documents and software. It provides informative guidance (guidance that is not normative and does not set requirements).
 
 This document is part of a series of technical and educational documents published by the [W3C Web Accessibility Initiative (WAI)](http://www.w3.org/WAI/) and available from the [WCAG 2 Overview](http://www.w3.org/WAI/intro/wcag.php).
```

Comment:
Change WCAG 2 Overview text and link destination to: `[WCAG2ICT Overview](https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/)`